api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -40,6 +40,8 @@
     private int jwksExpirySeconds;
     private String introspectionEndpointUri;
     private String userNameClaim;
+    private boolean skipTypeCheck;
+    private boolean notJwt;
```

Comment:
Should we set it to `false` as default? Or is it not needed?

Reply:
The default of the 'accessTokenIsJwt' is now `true`. No need for it to be explicitly set.